person/views.py

```diff
@@ -248,7 +248,9 @@ def godfather_create(request):
                 messages.error(request, str(v.args[0]))
         else:
             messages.error(request, 'Formulario con errores')
-    return render(request, 'godfather_form.html', {"form": form, "title": "Añadir Padrino"})
+
+    form = CreateNewGodFather()
+    return render(request, 'person/godfather/godfather_form.html', {"form": form, "title": "Añadir Padrino"})
```

Comment:
You have forgotten to change "godfather_form.html" to "form.html"

Reply:
Ups 🙃